codex/rest/api.nim

```diff
@@ -259,68 +104,58 @@ proc initRestApi*(node: CodexNodeRef, conf: CodexConf): RestRouter =
       return RestApiResponse.error(Http500)
 
   router.api(
-    MethodPost,
-    "/api/codex/v1/debug/chronicles/loglevel") do (
-      level: Option[string]) -> RestApiResponse:
-      ## Set log level at run time
-      ##
-      ## e.g. `chronicles/loglevel?level=DEBUG`
-      ##
-      ## `level` - chronicles log level
-      ##
-
-      without res =? level and level =? res:
-        return RestApiResponse.error(Http400, "Missing log level")
-
-      try:
-        {.gcsafe.}:
-          updateLogLevel(level)
-      except CatchableError as exc:
-        return RestApiResponse.error(Http500, exc.msg)
-
-      return RestApiResponse.response("")
+    MethodGet,
+    "/api/codex/v1/local") do () -> RestApiResponse:
```

Comment:
Shouldn't this go under `data/local`, you can also easily foresee a use for `data/local/{cid}` at some point, when the user wants to check if it has a particular cid.

Reply:
The limitation of the framework is that you can't have endpoints `/data/local` and `/data/{cid}` as it complains that the route already exists during initialization. There is a possible workaround, but we chose a simpler approach instead using `/local` endpoint. `/local/{cid}` can be supported with this approach as well.

Reply:
You just have to make `{cid}` an optional parameter for it to not complain.